package.json

```diff
@@ -55,33 +54,42 @@
 		"signal-exit": "^3.0.2",
 		"slice-ansi": "^3.0.0",
 		"string-length": "^3.1.0",
+		"type-fest": "^0.12.0",
+		"typescript": "^3.8.3",
 		"widest-line": "^3.1.0",
 		"wrap-ansi": "^6.2.0",
-		"yoga-layout-prebuilt": "^1.9.3"
+		"yoga-layout-prebuilt": "^1.9.5"
 	},
 	"devDependencies": {
-		"@babel/cli": "^7.8.3",
-		"@babel/core": "^7.8.3",
-		"@babel/plugin-proposal-class-properties": "^7.8.3",
-		"@babel/plugin-proposal-object-rest-spread": "^7.8.3",
-		"@babel/preset-react": "^7.8.3",
+		"@sindresorhus/tsconfig": "0.7.0",
+		"@types/is-ci": "^2.0.0",
+		"@types/lodash.throttle": "^4.1.6",
+		"@types/node": "^13.9.1",
 		"@types/react": "^16.9.19",
-		"ava": "^2.4.0",
+		"@types/react-reconciler": "^0.18.0",
```

Comment:
I may be wrong, but should types for modules in `dependencies` be included there as well, instead of `devDependencies`?

Reply:
I believe it's only necessary when the time is exposed as part of the API. I don't believe that react-reconciler is.

Reply:
I could be wrong, but I remember similar issues, where TS couldn't find typings for modules used internally by a dependency I installed.

Reply:
Ok, I'm happy to include them just to be sure.